blazor/pager/getting-started.md

```diff
@@ -0,0 +1,259 @@
+---
+layout: post
+title: Getting Started with Blazor Pager Component | Syncfusion
+description: Checkout and learn about getting started with Blazor Pager component in Blazor Server App and Blazor WebAssembly App.
+platform: Blazor
+control: Pager
+documentation: ug
+---
+
+# Getting Started with Blazor Pager Component
+
+This section briefly explains about how to include Blazor Pager component in your Blazor Server App and Blazor WebAssembly App using Visual Studio.
+
+## Prerequisites
+
+* [System requirements for Blazor components](https://blazor.syncfusion.com/documentation/system-requirements)
+
+## Create a new Blazor App in Visual Studio
+
+You can create **Blazor Server App** or **Blazor WebAssembly App** using Visual Studio in one of the following ways,
```

Comment:
You can create **Blazor Server App** or **Blazor WebAssembly App** using Visual Studio in one of the following ways: